app/src/api/orders/index.ts

```diff
@@ -90,8 +90,27 @@ export const useUpdateOrder = () => {
       return updatedOrder;
     },
     async onSuccess(_, { id }) {
-      await queryClient.invalidateQueries(['orders']);
-      await queryClient.invalidateQueries(['orders', id]);
+      await queryClient.invalidateQueries(["orders"]);
+      await queryClient.invalidateQueries(["orders", id]);
     },
   });
 };
+
+export const useCreateSolanaWallet = () => {
+  const queryClient = useQueryClient();
+  const { session } = useAuth();
+
+  return useMutation({
```

Comment:
hey Dan, i think this should be a working example of how the webapp would make requests to the web server. 

i haven't used expo web server so not totally sure how that works with auth middleware / adding users to the context but this *should* hit the createSolanaWallet endpoint. and then there should also be an invalidate query for whatever cache the get solana wallet  query uses.

Reply:
thanks!